.github/workflows/ci_tests_dev.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       matrix:
         python-version: [3.8]
         os: [ubuntu-20.04, macOS-10.15]
-        gmt_git_ref: [6.1, master]
+        gmt_git_ref: [master]
```

Comment:
Can you link to the issue that says 6.1 won't be updated anymore? I.e. that a 6.1.2 won't come out? I'd probably wait a few days just in case.

Still encountering some issues with the 'Q' projection at 0d702fd37d8ae48d9f04caf2e998b2ad43e95c2a, though it might just be a gridline/geographic type issue.

Reply:
It's mentioned at a few places:
https://github.com/GenericMappingTools/gmt/issues/3850#issuecomment-685984748, https://github.com/GenericMappingTools/gmt/issues/3850#issuecomment-685997806, 
https://github.com/GenericMappingTools/gmt/pull/4153.

The next release will be 6.2.0 in November (maybe, I think the GMT team decided to have a minor release every six months), unless there are critical issues in 6.1.1.

BTW, I'm OK keeping this PR unmerged for more days.